src/index.ts

```diff
@@ -111,7 +136,7 @@ interface ISQLite {
     status: 0 | 1;
     message?: string;
   };
-  transaction: (dbName: string, fn: (tx: Transaction) => boolean) => void;
```

Comment:
The boolean is idiomatic for a sync version, since everything has a sync and an async version, I think it is better to do the same for the transactions

Reply:
the try catch should prevent some run-away condition, I like it, but I would like to prefer to a pattern of explicit behaviors